src/main/kotlin/com/terraformation/backend/funder/FundingEntityService.kt

```diff
@@ -27,6 +33,8 @@ class FundingEntityService(
     private val dslContext: DSLContext,
     private val fundingEntityStore: FundingEntityStore,
     private val fundingEntityUserStore: FundingEntityUserStore,
+    private val userStore: UserStore,
+    private val publisher: ApplicationEventPublisher,
```

Comment:
If these two new arguments are flipped, the list of dependencies will remain in alphabetical order.